src/scenarios/shared/util.py

```diff
@@ -44,6 +44,12 @@ def uploadtokenpresent():
 def runninginlab():
     return environ.get('PERFLAB_INLAB') is not None
 
+def startupdir():
+    if sys.platform == 'win32':
+        return 'Startup'
+    else:
+        return 'startup'
```

Comment:
Instead of adding this here can we just fix the yml to build to `Startup`? I intended this but apparently got it wrong.

Reply:
Weird, I saw both having ```startup``` as publish output directory in yaml

Reply:
It is both `startup`. As Windows is case insensitive by default it hasn't mattered. So either a) change the yml to `Startup` or change this to `startup` for both platforms. I don't care which.